crates/turborepo-lib/src/commands/ls.rs

```diff
@@ -82,6 +84,9 @@ impl<'a> RepositoryDetails<'a> {
                 if !filtered_pkgs.contains(package_name) {
                     return None;
                 }
+                if matches!(package_name, PackageName::Root) {
```

Comment:
Not strictly speaking part of `--affected`, but in adding these tests, I noticed we had a bug